marketplace_builder/views/pages/how-to/migrations/migrating-data.liquid

```diff
@@ -0,0 +1,60 @@
+---
+converter: markdown
+metadata:
+  title: Migrating Data 
+  description: This guide will help you use migrations when you need to change the data or execute a piece of code outside of the regular application running cycle.
+slug: how-to/migrations/migrating-data
+searchable: true
+---
+This guide will help you use migrations when you need to change the data or execute a piece of code outside of the regular application running cycle, for example, when you want to seed data on each environment.
+
+## Requirements
+
+This is an advanced tutorial. To follow it, you should be familiar with basic Platform OS concepts, and the topics in the Get Started section. 
+
+* [How Platform OS Works](/how-platformos-works)
+* [Get Started](/get-started)
+
+## Steps
+
+Migrating data is a three-step process:
+
+1. Generate empty migration
+2. Implement migration
+3. Run migration
+
+### Step 1: Generate empty migration
+Generate an empty migration using the `marketplace-kit migrations` command: 
+
+<pre class="command-line" data-output="2-50" data-user="user" data-host="host"><code class="language-bash">
```

Comment:
`data-output` value (2-50) marks lines from - to where there is an output of a shell command - in this case there is no output at all (only user input), so you can remove the attribute altogether

Reply:
@pavelloz Thank you! Fixed.